src/views/Marketplace/SolutionView.vue

```diff
@@ -78,6 +78,11 @@
                 size="small"
                 class="p-0"
                 v-if="solution.isLaunched"
+                @click="
+                  openNewWindow(
+                    'https://www.azion.com/en/documentation/products/marketplace/integrations/'
+                  )
+                "
```

Comment:
Recomendo mover essa lógica para um método separado e chamá-lo aqui
```suggestion
                @click="
                  openNewWindow(
                    'https://www.azion.com/en/documentation/products/marketplace/integrations/'
                  )
                "
```